config/locales/idv/fr.yml

```
@@ -122,9 +122,9 @@ fr:
           you_entered: 'Tu as soumis:'
       sessions:
         exception: Une erreur interne s’est produite lors du traitement de votre demande.
-        fail_html: '<strong>Veuillez réessayer dans %{timeout}.</strong> Pour votre
-          sécurité, nous limitons le nombre de fois où vous pouvez tenter de
-          vérifier des informations personnelles en ligne.'
+        fail_html: 'Pour votre sécurité, nous limitons le nombre de fois où vous pouvez
+          tenter de vérifier des informations personnelles en ligne.
+          <strong>Réessayer dans %{timeout}.</strong>'
         heading: Nous ne trouvons pas de données qui correspondent à vos informations
```

Looks like the French heading wasn't updated (translation from the Jira ticket)? 
```suggestion
        heading: Nous n’avons pas trouvé de dossiers correspondant à vos informations personnelles
```